examples/oft_dreambooth/train_dreambooth.py

```diff
@@ -180,8 +180,8 @@ def parse_args(input_args=None):
 
     # oft args
     parser.add_argument("--use_oft", action="store_true", help="Whether to use OFT for parameter efficient tuning")
-    parser.add_argument("--oft_r", type=int, default=8, help="OFT rank, only used if use_oft is True")
-    parser.add_argument("--oft_alpha", type=int, default=32, help="OFT alpha, only used if use_oft is True")
+    parser.add_argument("--oft_r", type=int, default=0, help="OFT rank, only used if use_oft is True")
+    parser.add_argument("--oft_block_size", type=int, default=32, help="OFT block size, only used if use_oft is True")
```

Comment:
oft doesn't have `alpha`, it's `oft_block_size`, and align default value of `r` to `OFTConfig`